JASP-Desktop/JASP-Desktop.pro

```diff
@@ -210,8 +211,17 @@ unix {
     maketranslations.commands += $$GETTEXT_LOCATION/msgattrib --no-obsolete --no-location $$SOURCES_TRANSLATIONS/jasp.po -o $$SOURCES_TRANSLATIONS/jasp.po ;
     maketranslations.commands += $$GETTEXT_LOCATION/msgattrib --no-obsolete --no-location $$SOURCES_TRANSLATIONS/jasp_nl.po -o $$SOURCES_TRANSLATIONS/jasp_nl.po ;
 
+    #Create jasp_nl.qm
     maketranslations.commands += lrelease $$SOURCES_TRANSLATIONS/jasp_nl.po -qm $$RESOURCES_TRANSLATIONS/jasp_nl.qm ;
     maketranslations.commands += cp $$RESOURCES_TRANSLATIONS/*.qm $$RESOURCES_DESTINATION_TRANSLATIONS/ ;
+
+    #Create R-JASP.mo translation file. (Need to add GETTEXT location to PATH environment.)
+    maketranslations.commands += export SP=$PATH; export PATH=$PATH:$$GETTEXT_LOCATION;$$_R_HOME/Rscript $$PWD/../Tools/translate.R $$PWD/../JASP-Engine/JASP;export PATH=$SP ;
+
+    #Create R-JASPgraphs.mo translation file. (Need to add GETTEXT location to PATH environment.)
+    maketranslations.commands += export SP=$PATH; export PATH=$PATH:$$GETTEXT_LOCATION;$$_R_HOME/Rscript $$PWD/../Tools/translate.R $$PWD/../JASP-Engine/JASPgraphs;export PATH=$SP
```

Comment:
Do you really need to restore PATH? 
You could just do `PATH=$PATH:$$GETTEXT_LOCATION $$_R_HOME/Rscript $$PWD/../Tools/translate.R $$PWD/../JASP-Engine/JASPgraphs` and then it just uses it for that command.

And it would be even better if you first stored the `PATH=$PATH:$$GETTEXT_LOCATION` in a var or something instead of duplicating it.

Reply:
Yes, but I was afraid the PATH would grow every time you extend it. It uses a variable now and that works fine too.

Reply:
Yeah I understand, but that is why I am telling you.
If you change this to not include `export` it is temporary.
Like, the exact code I've shown you

Reply:
Just makes it simpler!

Reply:
And that way you do not actually ever touch PATH which I appreciate a lot, what if there is some mistake sometime later and you try to build JASP with a messed up PATH? Would be bad